model/BUILD.bazel

```diff
@@ -227,6 +228,8 @@ android_library(
     visibility = ["//visibility:public"],
     exports = [
         ":arguments_java_proto_lite",
+        ":profile_java_proto_lite",
```

Comment:
Don't add these, since we're removing the model library.

Reply:
Done